xray/test/test_plot.py

```diff
@@ -375,6 +375,28 @@ def test_diverging_color_limits(self):
         vmin, vmax = artist.get_clim()
         self.assertAlmostEqual(-vmin, vmax)
 
+    def test_default_title(self):
+        a = DataArray(np.random.randn(4, 3, 2, 1), dims=['a', 'b', 'c', 'd'])
+        self.plotfunc(a.isel(c=1))
+        title = plt.gca().get_title()
+        self.assertEqual('c = 1, d = 0', title)
+
+    def test_colorbar_label(self):
+        self.darray.name = 'testvar'
+        self.plotmethod()
+        alltxt = [t.get_text() for t in plt.gcf().findobj(mpl.text.Text)]
+        # Set comprehension not compatible with Python 2.6
+        alltxt = set(alltxt)
+        self.assertIn(self.darray.name, alltxt)
+
+    def test_no_labels(self):
+        self.darray.name = 'testvar'
+        self.plotmethod(add_labels=False)
+        alltxt = [t.get_text() for t in plt.gcf().findobj(mpl.text.Text)]
+        alltxt = set(alltxt)
+        for string in ['x', 'y', 'testvar']:
+            self.assertNotIn(string, alltxt)
+
```

Comment:
Since we added the `squeeze` functionality in this PR, let's add a test to make sure we can handle plotting data that is shaped in an edge case.  For example if `da.shape == (5, 1)`, I think we'll end up in the plot2d and will fail after the squeeze.